src/groundlight/unstableapi.py

```diff
@@ -0,0 +1,79 @@
+"""
+unstableapi.py
+
+This module is part of our evolving SDK. While these functions are designed to provide valuable functionality to enhance your projects, it's important to note that they are considered unstable. This means they may undergo significant modifications or potentially be removed in future releases, which could lead to breaking changes in your applications.
+"""
+import json
+from typing import Union
+
+from model import (
+    Channel,
+    Detector,
+    Rule,
+    RuleCreationInput,
+    SnoozeTimeUnit,
+    Verb,
+)
+from openapi_client.model.action import Action
+from openapi_client.model.condition import Condition
+from openapi_client.model.rule_creation_input import RuleCreationInput
+
+from groundlight import Groundlight
+
+
+class UnstableApi(Groundlight):
+    def __init__(self):
+        super().__init__()
+
+    def create_action(
+        self,
+        detector: Union[str, Detector],
+        rule_name: str,
+        channel: Union[str, Channel],
+        recipient: str,
+        *,
+        alert_on: Union[str, Verb] = "CHANGED_TO",
+        enabled: bool = True,
+        include_image: bool = False,
+        condition_parameters: Union[str, dict] = {"label": "NO", "num_consecutive_labels": "2"},
```

Comment:
Each condition type has it's own set of condition parameters that it needs to have correct here. So if you don't know the right parameters, you'll have some trouble. The parameters match what you see from the reef webpage. The proper way to handle this would be to model each of the condition types in the openapi spec, but that's a chunk of work for something that is sounds like we might end up changing

Reply:
That's okay for now. Although, why is it a union of `str | dict` -- isn't it always a dict?

Also, using `dict` defaults leads to tricky bugs (because it's mutable). Instead, we should default to `None`, and fill the default dict inside the function.

Reply:
It's a string if someone is crazy enough to use this from the command line

As for the other point, my favorite python bug rears its head again, thanks for catching this!